index.html

```diff
@@ -26,8 +26,125 @@ <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
         </ul>
-      </section>
-  </div>
-</main>
+     </section>
+    </div>
+  
+    <div class="cards">
+      <div class="card">
+        <img src="image/Snapshoot-Portfolio.png" alt="A photo of a website page">
+        <div class="card-content">
+          <div class="card-heading">Tonic</div>
+          <ul class="tool1">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>        
+          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+          <ul class="tool2">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <a href="#" class="detail-style2">See Project</a>
+        </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(1).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(2).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(3).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    </div>
+
+    <div class="about-section">
```

Comment:
- [ ] Since this div is supposed to be your main about container, I think it could be a nice idea if you use a semantic tag, in Html best practice, a semantic tag is more descriptive and professional than div, please change it into a `section` tag instead.